db/migrate/20200430041735_add_replay_count_to_game_purchases.rb

```diff
@@ -0,0 +1,20 @@
+class AddReplayCountToGamePurchases < ActiveRecord::Migration[6.0]
+  def change
+    add_column :game_purchases, :replay_count, :integer, null: false, default: 0
```

Comment:
Is this intentionally not nullable? The validation says null is allowed.

Reply:
@connorshea the reason why the validation allows null is because postgres will be smart and substitute the default value of 0 for null values. So it's stupid to validate it on the rails level, because it's fine to not set that attribute, the DB will set it to `0` anyway.